vllm/tracing.py

```diff
@@ -92,6 +92,9 @@ class SpanAttributes(BaseSpanAttributes):
     LLM_LATENCY_TIME_IN_QUEUE = "gen_ai.latency.time_in_queue"
     LLM_LATENCY_TIME_TO_FIRST_TOKEN = "gen_ai.latency.time_to_first_token"
     LLM_LATENCY_E2E = "gen_ai.latency.e2e"
+    LLM_LATENCY_TIME_IN_SCHEDULER = "gen_ai.latency.time_in_scheduler"
+    LLM_LATENCY_TIME_IN_MODEL_FORWARD = "gen_ai.latency.time_in_model_forward"
+    LLM_LATENCY_TIME_IN_MODEL_EXECUTE = "gen_ai.latency.time_in_model_execute"
```

Comment:
can you update the comments for these traces in this file? basically the same comment as 

```
    # Total time spent in the forward pass for this across all workers
    model_forward_time: Optional[float] = None

    # Total time spent in the model execute function. This will include model
    # forward, block/sync across workers, cpu-gpu sync time and sampling time.
    model_execute_time: Optional[float] = None
```

Reply:
Done.